apps/web/src/components/PaginatedTable/helpers/transactions.tsx

```diff
@@ -0,0 +1,187 @@
+  >[]
+) => {
+  const headers = [
+    {
+      cells: [
+        {
+          item: "Blob Versioned Hash",
+        },
+        {
+          item: "Size",
+        },
+      ],
+      className: "dark:border-border-dark/20",
+      sticky: true,
+    },
+  ];
+
+  const rows = blobs
+    ? blobs.map((b) => ({
+        cells: [
+          {
+            item: b.versionedHash,
+          },
+          {
+            item: (
+              <div className="flex gap-2 text-xs">
+                <span>{formatBytes(b.size)}</span>
+              </div>
+            ),
+          },
+        ],
+        className: "dark:border-border-dark/10",
+      }))
+    : undefined;
+
+  return (
+    <Table
+      className="mb-4 mt-2 max-h-[420px] rounded-lg bg-primary-50 px-8 dark:bg-primary-800"
+      size="xs"
+      alignment="left"
+      variant="transparent"
+      headers={headers}
+      rows={rows}
+    />
+  );
+};
+
+export const getTransactionsTableRows = (transactions?: Transaction[]) =>
+  transactions
+    ? transactions.map((t: Transaction) => {
+        const {
+          hash,
+          from,
+          to,
+          blobs,
+          rollup,
+          blockNumber,
+          blobGasBaseFee,
+          blobGasMaxFee,
+          blockTimestamp,
+        } = t;
+        return {
+          cells: [
+            {
+              item: shortenAddress(hash, 6),
+            },
+            {
+              item: (
+                <div className="text-contentTertiary-light dark:text-contentTertiary-dark">
+                  <Link href={buildBlockRoute(blockNumber)}>{blockNumber}</Link>
+                </div>
+              ),
+            },
+            {
+              item: (
+                <div className="min-w-52 whitespace-break-spaces">
+                  {formatTimestamp(blockTimestamp)}
+                </div>
+              ),
+            },
+            {
+              item: (
+                <Link href={buildAddressRoute(from)}>
+                  {shortenAddress(from, 6)}
+                </Link>
+              ),
+            },
+            {
+              item: (
+                <Link href={buildAddressRoute(to)}>
+                  {shortenAddress(to, 6)}
+                </Link>
+              ),
+            },
+            {
+              item: rollup ? <RollupIcon rollup={rollup} /> : <></>,
+            },
+            {
+              item: (
+                <div className="truncate">
+                  <EtherUnitDisplay amount={blobGasBaseFee} />
+                </div>
+              ),
+            },
+            {
+              item: (
+                <div className="truncate">
+                  <EtherUnitDisplay amount={blobGasMaxFee} />
+                </div>
+              ),
+            },
+            {
+              item: (
+                <div className="truncate">
+                  <EtherUnitDisplay amount={blobGasMaxFee} />
+                </div>
```

Comment:
Let's fix the ether unit display to `Gwei` for the blob gas base and max fee, and the gras price as showing different ones base on the scaling factor can be confusing.

Reply:
Applied `Gwei` to all the ether displays